PaigeVannelli/shine
# Feature/new post form styles

## Pull Request
### Description
* Is this a feature or a fix?
Feature!

* Where should the reviewer start?
Please start on the NewPostForm

* What functionalities do these changes affect? Why were these changes made? What was the motivation behind these changes?
Adds styling to the newPostForm page as well as the Loading page. Also adds error handling to the NewPostForm page. The user should not be able to submit the form unless both fields are filled out with at least one character. The button, when deactivated, should be grey. When the user can submit the form, the button turns yellow. The X button at the top of the page should lead back to the home page when pressed at any time. Upon returning to the newPostForm page after clicking the X, the form should be empty and ready to go. 

Form should look good across all phone sizes

* What outside features or research did you use to implement this fix?
N/A

* How should this be tested?
Please pull down the code and click around to make sure everything is functioning. 


closes #17 
closes #31
